CRM/Sumfields/Form/SumFields.php

```diff
@@ -29,10 +30,14 @@ function buildQuickForm() {
       preg_match('/^(scheduled|running|success|failed):([0-9 :\-]+)$/', $apply_settings_status, $matches);
       $date = $matches[2];
       switch($matches[1]) {
-        case 'scheduled':
+        case 'scheduled' and $process_type == 'via_triggers':
           $display_status = ts("Setting changes were saved on %1, but not yet applied; they should be applied shortly.", array(1 => $date, 'domain' => 'net.ourpowerbase.sumfields'));
           $status_icon = 'fa-hourglass-start';
           break;
+        case 'scheduled' and $process_type == 'via_cron':
+          $display_status = ts("Setting changes were saved on %1, but not yet applied; they should be applied on next cron run.", array(1 => $date, 'domain' => 'net.ourpowerbase.sumfields'));
+          $status_icon = 'fa-hourglass-start';
+          break;
         case 'running':
```

Comment:
I'm not sure we need this change. If the change is scheduled, it will be applied on the next cron job - regardless of whether the process type is via triggers or not. I think we can simply leave this out and let the user know that the changes will be applied shortly.

Reply:
True, basically I wanted to separate the messages a little bit, but ... makes no sense :(